pkg/webrtctransport.go

```diff
@@ -194,26 +120,36 @@ func (p *WebRTCTransport) CreateOffer() (webrtc.SessionDescription, error) {
 	if err != nil {
 		return webrtc.SessionDescription{}, err
 	}
-
 	parsed := sdp.SessionDescription{}
```

Comment:
For some reason, using the ```offer.Unmarshal``` will bring an incorrect parsed SDP, and I'm not sure why.